internal/deploy/kustomize/daily/base/websphere-liberty-deployment.yaml

```diff
@@ -1,40 +1,57 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: websphere-liberty-controller-manager
   labels:
-    control-plane: websphere-liberty-controller-manager
     app.kubernetes.io/instance: websphere-liberty-operator
     app.kubernetes.io/name: websphere-liberty-operator
+    control-plane: websphere-liberty-controller-manager
+  name: websphere-liberty-controller-manager
+  namespace: default
 spec:
+  replicas: 1
   selector:
     matchLabels:
+      app.kubernetes.io/instance: websphere-liberty-operator
+      app.kubernetes.io/name: websphere-liberty-operator
       control-plane: websphere-liberty-controller-manager
-  replicas: 1
   template:
     metadata:
+      annotations:
+        productChargedContainers: ALL
+        productID: cb1747ecb831410f88006195f024183f
+        productMetric: FREE
+        productName: WebSphere Liberty Operator
       labels:
+        app.kubernetes.io/instance: websphere-liberty-operator
+        app.kubernetes.io/name: websphere-liberty-operator
         control-plane: websphere-liberty-controller-manager
-      annotations:
-        productID: "cb1747ecb831410f88006195f024183f"
-        productName: "WebSphere Liberty Operator"
-        productMetric: "FREE"
-        productChargedContainers: "ALL"
     spec:
-      hostIPC: false
-      hostPID: false
-      hostNetwork: false
-      securityContext:
-        runAsNonRoot: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
       containers:
-      - command:
-        - /manager
-        args:
-        - "--health-probe-bind-address=:8081"
-        - "--metrics-bind-address=127.0.0.1:8080"
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
         - --enable-leader-election
-        image: cp.stg.icr.io/cp/websphere-liberty-operator@sha256:b2b0f07db844e0b3970cfdf82f87a0ce330ee38f34df6926c09535b8ecbd2832
-        name: manager
+        command:
+        - /manager
+        env:
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: OPERATOR_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: icr.io/cpopen/websphere-liberty-operator:daily
```

Comment:
This image isn't currently available, as we haven't released yet. The previous files were referencing an sha256 tag for an image on staging. Could you check what the correct image to reference here is? 
The install breaks as things are at the moment